tiledb/ml/models/_base.py

```diff
@@ -253,7 +255,7 @@ def _load_tensorboard(self, timestamp: Optional[Timestamp] = None) -> None:
                     f.write(file_bytes)
 
     @staticmethod
-    def is_v1(array: tiledb.Array) -> bool:
-        if array.schema.domain.size < np.iinfo(np.uint64).max - 1025:
-            return True
-        return False
+    def is_v1() -> bool:
```

Comment:
Perhaps rename to `use_legacy_schema` or somesuch; `v1` and `v2` are misleading given the actual version is 0.8/0.9 etc. 

Similarly `__load_v1`, `__load__v2`: Rename `__load_v1` to `__load_legacy` and move `__load__v2` into `load`